packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart

```diff
@@ -43,25 +44,24 @@ class _PropertiesList extends StatelessWidget {
       builder: (context, args, _) {
         return args.isEmpty
             ? const Center(
-              child: Text('No widget properties at current cursor location.'),
+              child: Text(
+                'No widget properties at the current cursor location.',
+              ),
             )
             : Column(
-              children: [
-                for (final property in args)
-                  ...<Widget>[
-                    _EditablePropertyItem(property: property),
-                  ].joinWith(const PaddedDivider.noPadding()),
-              ],
+              children: <Widget>[
+                ...args.map((arg) => _EditablePropertyItem(argument: arg)),
+              ].joinWith(const PaddedDivider.noPadding()),
```

Comment:
can we get rid of the outer list and just write:
```
children: args.map((arg) => _EditablePropertyItem(argument: arg)).toList().joinWith(const PaddedDivider.noPadding()),

Reply:
I think the outer list (or a cast) is necessary so that the list is of type `Widget` instead of type `_EditablePropertyItem`. Otherwise calling `.joinWith` results in the error that `PaddedDivider` is not type `_EditablePropertyItem`.